shared/wallets/sep7-confirm/container.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react'
+import * as RouteTreeGen from '../../actions/route-tree-gen'
+import * as WalletsGen from '../../actions/wallets-gen'
+import * as Constants from '../../constants/wallets'
+import * as Container from '../../util/container'
+import SEP7Confirm from '.'
+
+const mapStateToProps = (state: Container.TypedState) => {
+  const error = state.wallets.sep7ConfirmError
+  if (error) {
+    return {error}
+  }
+
+  const {
+    amount,
+    availableToSendFiat,
+    availableToSendNative,
+    assetCode,
+    assetIssuer,
+    callbackURL,
+    displayAmountFiat,
+    memo,
+    memoType,
+    message,
+    operation,
+    originDomain,
+    recipient,
+    summary,
+    xdr,
+  } = state.wallets.sep7ConfirmInfo
```

Comment:
I'd prefer just sticking this `sep7ConfirmInfo` in to the returned object here and destruct in `mergeProps` to minimize work in `mapStateToProps`